env0/data_project_test.go

```diff
@@ -0,0 +1,60 @@
+package env0
+
+import (
+	"testing"
+
+	"github.com/env0/terraform-provider-env0/client"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+)
+
+var project = client.Project{
+	Id:          "id0",
+	Name:        "my-project-1",
+	CreatedBy:   "env0",
+	Role:        "role0",
+	Description: "A project's description",
+}
```

Comment:
that's bad..
`project` is now a global variable in `env0` package.
i got the same reject from @yaronya

Reply:
Yeah I'm still thinking in JS modules ...
Anyway I did https://github.com/env0/terraform-provider-env0/pull/98/commits/35233e15a653a1aba41424c3c56375833f75d9b8 which solves it and also makes a nicer structure.
WDYT ?

Reply:
look good, just wondering how it behaves with `t.Run` because we didn't use it before, whats the difference

Reply:
nvm it sounds great
https://golang.org/pkg/testing/#hdr-Subtests_and_Sub_benchmarks